example/app/components/AddTaskForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  Pressable,
+  Platform,
+  StyleSheet,
+} from 'react-native';
+
+import {buttonStyles} from '../styles/button';
+import colors from '../styles/colors';
+import {shadows} from '../styles/shadows';
+
+interface AddTaskFormProps {
```

Comment:
In general I've seen props being declared using `type` more often than `interface` [these days](https://stackoverflow.com/a/59561874/503899). In any case I think we should keep this consistent across the components.